modules/ROOT/pages/to-request-vpc-connectivity.adoc

```diff
@@ -33,6 +33,10 @@ To avoid processing delays, ensure that you complete all fields on both tabs.
 The MuleSoft Support team updates the VPN configuration using the details provided in the Anypoint VPC discovery form.
 Support updates your ticket to notify you when your request is complete.
 
+== Configure Transit Gateway Attachment Connections
+
+For transit gateway attachments, follow the self-service steps in 
```

Comment:
"For transit gateway attachments" is not really a topic; can you make this more specific?